packages/block-library/src/quote/index.js

```diff
@@ -49,6 +49,7 @@ export const settings = {
 	description: __( 'Maybe someone else said it better -- add some quoted text.' ),
 	icon: <svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="none" d="M0 0h24v24H0V0z" /><g><path d="M19 18h-6l2-4h-2V6h8v7l-2 5zm-2-2l2-3V8h-4v4h4l-2 4zm-8 2H3l2-4H3V6h8v7l-2 5zm-2-2l2-3V8H5v4h4l-2 4z" /></g></svg>,
 	category: 'common',
+	keywords: [ __( 'blockquote' ) ],
```

Comment:
@ZebulanStanphill This probably shouldn't be translatable?

Reply:
Oh, why not? Wouldn’t translating it improve autocomplete in other locales?

Reply:
I think the point is to match the HTML element here, with translations it'd mean that it can't be discovered by using "blockquote". Specially given there's no translation context, I'd imagine in Spanish this would be translated as "cita".

Reply:
I just made this translatable because keywords for other blocks were translatable, e.g. "movie" for the Video block. I also thought blockquote was a term that was used outside of the context of HTML for any quote shown in a container to make it stand out from the rest of the content. Am I mistaken on that?

Reply:
It's more of a general question, for sure. We don't translate `hr` for separator, but we do translate `separator`. This one is a bit blurry.